andela/bp-esa-frontend
# #162751892: Engagement trend visualisation

#### What does this PR do?
- Displays engagement trend (onBoarding and offBoarding) on a graph

#### Description of Task to be completed?
- Displays engagement trend (onBoarding and offBoarding) on a graph

#### How should this be manually tested?
- Clone the frontend repo and the backend repo
- Run both frontend and backend repo
- log in to the frontend and click on dashboard
- you will be able to see a graph for onBoarding and offBoarding
#### Any background context you want to provide?
- This PR uses dummyData
#### What are the relevant pivotal tracker stories?
[162751892](https://www.pivotaltracker.com/story/show/162751892)
#### Postman Documentation
- N/A
#### Screenshots (If applicable)
<img width="954" alt="Screenshot 2019-09-05 at 11 27 01" src="https://user-images.githubusercontent.com/29975767/64325068-27d61700-cfd0-11e9-9778-e9862bc68b80.png">
